From 78c70ff55bb8e5ae4778b64faa4b31af7dfff546 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Fri, 1 Jan 2021 17:34:08 +0100
Subject: [PATCH 13/21] Fix prerequisites of stamps/control

debian/control.cross.in is used as input even when TARGET= and
with_cross=no.
---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index faaf2a3..9c0e136 100755
--- a/debian/rules
+++ b/debian/rules
@@ -588,7 +588,7 @@ endif
 
 ###############################################################################
 
-stamps/control: debian/control.in $(if $(TARGET),debian/control.cross.in) $(if $(filter yes, $(with_cross)),debian/control.cross.in)
+stamps/control: debian/control.in debian/control.cross.in
 ifneq (,$(TARGET))
 	sed -n -e 's/@dpkg_dev@/$(DPKG_DEV)/;/^Package: binutils-for-build$$/q;p' \
 		< debian/control.in > debian/control
-- 
2.30.1

